.buildkite/pipeline.yml

```diff
@@ -429,6 +429,22 @@ steps:
           slurm_gpus: 1
           slurm_mem: 16G
 
+      - label: ":computer: test restart"
+        command: >
+          julia --color=yes --project=examples test/restart.jl MANYTESTS
```

Comment:
```suggestion
          julia --color=yes --project=examples test/restart.jl --MANYTESTS true
```
Just a reminder: we'll need to update this too

Reply:
Yes, thank you